packages/core/src/index.js

```diff
@@ -1,57 +1,59 @@
-export { default as Absolute } from './Absolute'
-export { default as Avatar } from './Avatar'
-export { default as BackgroundImage } from './BackgroundImage'
-export { default as Badge } from './Badge'
-export { default as Banner } from './Banner'
-export { default as BlockLink } from './BlockLink'
-export { default as Box } from './Box'
-export { default as Breadcrumbs } from './Breadcrumbs'
-export { default as Button } from './Button'
-export { default as Card } from './Card'
-export { default as CloseButton } from './CloseButton'
-export { default as Container } from './Container'
-export { default as Divider } from './Divider'
-export { default as Flag } from './Flag'
-export { default as Flex } from './Flex'
-export { default as Heading } from './Heading'
-export { default as Hide } from './Hide'
-export { default as Hug } from './Hug'
-export { default as Icon } from './Icon'
-export { default as IconButton } from './IconButton'
-export { default as IconField } from './IconField'
-export { default as Image } from './Image'
-export { default as Input } from './Input'
-export { default as InputField } from './FormField'
-export { default as FormField } from './FormField'
-export { default as InputGroup } from './InputGroup'
-export { default as Label } from './Label'
-export { default as Link } from './Link'
-export { default as PlaceholderImage } from './PlaceholderImage'
-export { default as Radio } from './Radio'
-export { default as RatingBadge } from './RatingBadge'
-export { default as Relative } from './Relative'
-export { default as Select } from './Select'
-export { default as SrOnly } from './SrOnly'
-export { default as Stamp } from './Stamp'
-export { default as Stepper } from './Stepper'
-export { default as Text } from './Text'
-export { default as TextArea } from './TextArea'
-export { default as Tooltip } from './Tooltip'
-export { default as ToggleBadge } from './ToggleBadge'
-export { default as Truncate } from './Truncate'
-export { default as Checkbox } from './Checkbox'
+export { Absolute } from './Absolute'
+export { Avatar } from './Avatar'
+export { BackgroundImage } from './BackgroundImage'
+export { Badge } from './Badge'
+export { Banner } from './Banner'
+export { BlockLink } from './BlockLink'
 
-export { default as theme } from './theme'
-export { default as ThemeProvider } from './ThemeProvider'
-export { default as createTheme } from './createTheme'
+export * from './Box'
```

Comment:
Why is this different from the others?

Reply:
it's not anymore. this was leftover from an earlier iteration and i missed it